consensus/src/counters.rs

```diff
@@ -322,12 +228,6 @@ pub static NUM_TXNS_PER_BLOCK: Lazy<Histogram> = Lazy::new(|| {
     .unwrap()
 });
 
-/// Histogram of per-transaction execution time of non-empty blocks
-/// (calculated as the overall execution time of a block divided by the number of transactions).
-pub static TXN_EXECUTION_DURATION_S: Lazy<DurationHistogram> = Lazy::new(|| {
```

Comment:
Are you removing these counters (txn execution duration / block execution duration) because there are similar counters at the level of executor?

Reply:
txn execution duration is derivable from block execution + txn per block, block execution is still tracked.